src/header.tsx

```diff
@@ -37,11 +37,13 @@ const Header: React.SFC = () => {
                     <Link color="inherit" component={RouterLink} to="/">Home </Link>
                     <Link color="inherit" component={RouterLink} to="/acquire">Acquire</Link>
                 </Typography>
-                <Button color="inherit">Login</Button>
+                <Typography variant="h5" className={classes.title}>
```

Comment:
This should remain a button, with the alignment it had. I think this needs to change before anything is merged. The button can also use RouterLink and the to can't it? We should figure out the syntax if it is a little different.

Reply:
button stored - not sure why I removed it in the first place (possibly just to play around with the text) https://github.com/bluesky/bluesky-webclient/pull/21/commits/a0f552b80bfccc90565b4147a7f286001feeaaa9